RELEASE_PROCEDURE.rst

```diff
@@ -50,10 +54,13 @@ pika follows `Semantic Versioning <https://semver.org>`_.
    You'll need to set ``GITHUB_TOKEN`` or both ``GITHUB_USER`` and
    ``GITHUB_PASSWORD`` for the hub release command.
 
-#. Merge release branch into ``main``.
+#. Merge release branch into ``main`` (with --no-ff).
```

Comment:
Why `--no-ff`?

Reply:
`--no-ff` creates an explicit merge commit and is a bit more obvious about the merge.